plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/DeployJob.java

```diff
@@ -85,13 +80,11 @@
   private final boolean includeOptionalConfigurationFiles;
   private final CollectingLineListener errorCollectingLineListener;
 
-  public StandardDeployJob(IProject project,
-                           Credential credential,
-                           IPath workDirectory,
-                           ProcessOutputLineListener stagingStdoutLineListener,
-                           ProcessOutputLineListener stderrLineListener,
-                           DefaultDeployConfiguration deployConfiguration,
-                           boolean includeOptionalConfigurationFiles) {
+  public DeployJob(IProject project, Credential credential, IPath workDirectory,
+      ProcessOutputLineListener stagingStdoutLineListener,
```

Comment:
Done. From the API design perspective, I think it's better to hide any `CloudSdk` related things such as `ProcessOutputLineListener` and `DefaultDeployConfiguration`, but that'd call for another opportunity for refactoring.